modules/ROOT/pages/ldap/ldap-connector-reference.adoc

```diff
@@ -19,8 +19,9 @@ Release Notes: xref:release-notes::connector/ldap-connector-release-notes-mule-4
 |Name | String | The name for this configuration. Connectors reference the configuration with this name. | | x
 | Connection a| * Basic Configuration
 * TLS Configuration
- | The connection types that can be provided to this configuration. | | x
-| Expiration Policy a| ExpirationPolicy |  Configures the minimum amount of time that a dynamic configuration instance can remain idle before the runtime considers it eligible for expiration. This does not mean that the platform expires the instance at the exact moment that it becomes eligible. The runtime actually purges the instances when it sees fit. |  |
+ | The connection types to provide to this configuration. | | x
+| Expiration Policy a| ExpirationPolicy |  Configures the minimum amount of time that a dynamic configuration instance can remain idle before Mule considers it eligible for expiration. This does not mean that the platform expires the instance at the exact moment that it becomes eligible. Mule actually purges the instances when it sees fit. |  |
```

Comment:
We have alternative text for this field description here: https://docs.google.com/spreadsheets/d/1FzjhPtRWNHmaNK_8JbXbkbblZiul6zELQ2I4Hys0_UI/edit?usp=sharing